<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MyGraphicsView" name="graphicsView">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Медиа</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionNext_Frame"/>
    <addaction name="actionPrev_Frame"/>
    <addaction name="separator"/>
    <addaction name="actionClear_marks"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrev_Frame"/>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="actionPause"/>
   <addaction name="actionNext_Frame"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/document-open-symbolic.svg</normaloff>:/icons/document-open-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zoom-in-symbolic.svg</normaloff>:/icons/zoom-in-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In </string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zoom-out-symbolic.svg</normaloff>:/icons/zoom-out-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/media-playback-start-symbolic.svg</normaloff>:/icons/media-playback-start-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/media-playback-pause-symbolic.svg</normaloff>:/icons/media-playback-pause-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/media-playback-stop-symbolic.svg</normaloff>:/icons/media-playback-stop-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionNext_Frame">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/media-skip-forward-symbolic.svg</normaloff>:/icons/media-skip-forward-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Next Frame</string>
   </property>
  </action>
  <action name="actionPrev_Frame">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/media-skip-backward-symbolic.svg</normaloff>:/icons/media-skip-backward-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>Prev Frame</string>
   </property>
  </action>
  <action name="actionClear_marks">
   <property name="text">
    <string>Clear marks</string>
   </property>
  </action>
  <action name="actionZoom_1_1">
   <property name="text">
    <string>Zoom 1:1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
